### DISCUSSION TITLE
Features as substitute for ZOTILO (under ZOTERO 6)

### DISCUSSION THREAD

**Discussion by guido-gitto** [windingwind/zotero-actions-tags#364] [General]
As I seem to be a complete layman but really would like to add some features to the new version of ZOTERO, I would be very very very grateful, if any one of you would be kind enough to help me with a set that I could simply copy and paste into the »data«-field? The following actions I would like to integrate with my own shortcuts to speed up my work:

(a) »create link of marked item« (such as titles, notes etc.) to connect data inside my database
(b) »create note to this title«
(c) »copy marked item«

Best regards from Germany, where it is so hot, that my English is even worse than usually (cheap excuse, I know) :)
Guido

**Comment by windingwind** [comment]
> a

seems to be #115, if I understand correctly

> b

Do you mean to create a child note for an item?

> c

I'm not sure what the `marked` item is (do you mean selected item?). and what `copy` means here. Where would you expect to paste the copied content, a text editor, or Zotero itself?

**Reply by guido-gitto** [reply]
Thank you so very much! You set me on the right track, and it finally works! Just to answer your kind questions:

(b) simply meant »create a note for select item« - and it works fine, too.
(c) Yes, what I was trying to say is »copy this Zotero item« (e.g. a note)

I wish I could do more than say »Dankeschön«.

**Comment by mjthoraval** [comment]
> (b) »create note to this title«

> »create a note for select item«

This is already a Zotero shortcut:
![image](https://github.com/user-attachments/assets/e1b32b97-a304-415c-aaae-59a95b97b24c)
https://www.zotero.org/support/kb/keyboard_shortcuts#adding_items_to_your_zotero_library

And you can change it in the Advanced Settings:
![image](https://github.com/user-attachments/assets/76ef6d25-738a-4628-b8b7-764bcd368dc8)

**Reply by guido-gitto** [reply]
Thank you so much!

**Comment by thalient-ai** [comment]
These scripts will let you copy and paste item fields, tags, creators, etc
[Zutilo Copy](https://github.com/windingwind/zotero-actions-tags/discussions/383)
[Zutilo Paste](https://github.com/windingwind/zotero-actions-tags/discussions/384)
